app/views/directives/add-config-to-application.html

```diff
@@ -39,7 +39,7 @@ <h3>Add to Application</h3>
           <div class="has-warning" ng-if="ctrl.hasInvalidEnvVars">
             <div class="help-block">
               <span class="pf-icon pficon-warning-triangle-o" aria-hidden="true"></span>
-              Some of the keys for {{ctrl.apiObject.kind | humanizeKind}} <strong>{{ctrl.apiObject.metadata.name}}</strong> are not valid environment variable names and will not be added.
+              Some of the keys for {{ctrl.apiObject.kind | humanizeKind}} <strong>{{ctrl.apiObject.metadata.name}}</strong> are not valid environment variable names.
```

Comment:
The message was saying just the invalid keys won't be added, not that the secret won't be added. But maybe the wording was confusing.

I think we still need to tell the user what will happen. How about...?

```html
Some of the keys for {{ctrl.apiObject.kind | humanizeKind}} <strong>{{ctrl.apiObject.metadata.name}}</strong> are not valid environment variable names.
Only {{ctrl.apiObject.kind | humanizeKind}} keys with valid names will be added as environment variables.
```

cc @jwforres

Reply:
that sounds good to me :+1:

Reply:
So I like @spadget's message better generally than what is there.  My point in the bug is that we say "Some" even when its really "All"

Reply:
If we say

```
{{ctrl.apiObject.kind | humanizeKind}} <strong>{{ctrl.apiObject.metadata.name}}</strong> contains keys that are not valid environment variable names.
Only {{ctrl.apiObject.kind | humanizeKind}} keys with valid names will be added as environment variables.
```

then one message works generally

Reply:
will need to update that first humanized string so the first letter is capitalized, i.e. "Config map"

Reply:
That message is better 👍 

 Will need `upperFirst` to capitalize the first word in the sentence.

```html
{{ctrl.apiObject.kind | humanizeKind | upperFirst}} <strong>{{ctrl.apiObject.metadata.name}}</strong> ...
```

Reply:
IIRC the capitalize option on humanizeKind filter capitalizes both words?  if so then you can just wrap this humanizeKind filter call in parens and pass it to _.capitalize

Reply:
@jwforres We were making the same comment at the same time :)

I think `upperFirst` filter should do what we want.